src/js/utils/is-in-viewport.js

```diff
@@ -1,11 +1,14 @@
 // https://stackoverflow.com/a/7557433
-module.exports = function isElementInViewport (el) {
+function isElementInViewport (el, win=window,
+                              htmlEl=document.documentElement) {
```

Comment:
When I see `htmlEl`, I don't really differentiate it from `el`. Should it be `docEl` or something?